quickstartKotlin/src/main/java/com/twilio/video/quickstart/kotlin/VideoActivity.kt

```diff
@@ -695,6 +692,54 @@ class VideoActivity : AppCompatActivity() {
         muteActionFab.setOnClickListener(muteClickListener())
     }
 
+    /*
+     * Show the current available audio devices.
+     */
+    private fun showAudioDevices() {
+        val selectedDevice = audioDeviceSelector.selectedAudioDevice
+        val availableAudioDevices = audioDeviceSelector.availableAudioDevices
+
+        if (selectedDevice != null) {
+            val selectedDeviceIndex = availableAudioDevices.indexOf(selectedDevice)
+            val audioDeviceNames = ArrayList<String>()
+
+            for (a in availableAudioDevices) {
+                audioDeviceNames.add(a.name)
+            }
+
+            AlertDialog.Builder(this)
+                    .setTitle(R.string.room_screen_select_device)
+                    .setSingleChoiceItems(
+                            audioDeviceNames.toTypedArray<CharSequence>(),
+                            selectedDeviceIndex
+                    ) { dialog, index ->
+                        dialog.dismiss()
+                        val selectedAudioDevice = availableAudioDevices[index]
+                        updateAudioDeviceIcon(selectedAudioDevice)
+                        audioDeviceSelector.selectDevice(selectedAudioDevice)
+                    }.create().show()
+        }
+    }
+
+    /*
+     * Update the menu icon based on the currently selected audio device.
+     */
+    private fun updateAudioDeviceIcon(selectedAudioDevice: AudioDevice?) {
+        var audioDeviceMenuIcon = R.drawable.ic_phonelink_ring_white_24dp
+
+        if (selectedAudioDevice is AudioDevice.BluetoothHeadset) {
```

Comment:
@Alton09 I could not figure out why only the bluetooth headset required `AudioDevice.BluetoothHeadset` as opposed to just `BluetoothHeadset`. Thoughts?

Reply:
You should be able to import BluetoothHeadset directly with
```
import com.twilio.audioswitch.selection.AudioDevice.BluetoothHeadset
```
I just tried it on your branch and it works. Also, I recommend using a when expression opposed to if else, which would look like this...
```
private fun updateAudioDeviceIcon(selectedAudioDevice: AudioDevice?) {
    val audioDeviceMenuIcon = when(selectedAudioDevice) {
        is BluetoothHeadset -> R.drawable.ic_bluetooth_white_24dp
        is WiredHeadset -> R.drawable.ic_headset_mic_white_24dp
        is Speakerphone -> R.drawable.ic_volume_up_white_24dp
        else -> R.drawable.ic_phonelink_ring_white_24dp
    }

    audioDeviceMenuItem.setIcon(audioDeviceMenuIcon)
}
```

Reply:
You should be able to import BluetoothHeadset directly with
```
import com.twilio.audioswitch.selection.AudioDevice.BluetoothHeadset
```
I just tried it on your branch and it works. Also, I recommend using a when expression opposed to if else, which would look like this...
```
private fun updateAudioDeviceIcon(selectedAudioDevice: AudioDevice?) {
    val audioDeviceMenuIcon = when(selectedAudioDevice) {
        is BluetoothHeadset -> R.drawable.ic_bluetooth_white_24dp
        is WiredHeadset -> R.drawable.ic_headset_mic_white_24dp
        is Speakerphone -> R.drawable.ic_volume_up_white_24dp
        else -> R.drawable.ic_phonelink_ring_white_24dp
    }

    audioDeviceMenuItem.setIcon(audioDeviceMenuIcon)
}
```